addons/crm_iap_mine/static/src/js/crm_lead_gen_element.js

```diff
@@ -0,0 +1,22 @@
+import { patch } from "@web/core/utils/patch";
+import {
+    LeadGenerationDropdown,
+    MODULE_STATUS
+} from "@crm/components/lead_generation_dropdown/lead_generation_dropdown";
+
+patch(LeadGenerationDropdown.prototype, {
+    setup() {
+        super.setup();
+        const iapElement = this.state.dropdownContentElements.find(
+            element => element.moduleXmlId === 'base.module_crm_iap_mine'
+        );
+        iapElement.onClick = () => this.openLeadGenerationForm();
+        iapElement.status = MODULE_STATUS.INSTALLED;
+        iapElement.model = "crm.iap.lead.mining.request";
```

Comment:
Maybe `Object.assign(iapElement, {});` in case there's other stuff to add later, seems slightly more readable. (not a major issue either)